src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -468,12 +468,6 @@ InterpreterSelectQuery::InterpreterSelectQuery(
         }
     }
 
-    /// Set skip_unavailable_shards to true only if it wasn't disabled explicitly
```

Comment:
What was changed since putting this comment? Was it done by mistake?

Reply:
This check doesn't consider whether parallel replicas will be actually used or not. And the parallel replicas will not be used with `clusterAllReplicas()` since it considers each replica as a shard with 1 replica in it, i.e. not applicable for parallel replicas